File: Assignment_24/program1.c
// Write a program which accept string from user and count number of capital characters.

///////////////////////////////////////////////////////////////////////////////
// File Name   :  program1.c
// Description :  Accept string from user and count number of capital characters
// Autor       :  Ankita Anil Patil
// Date        :  06/06/2025
///////////////////////////////////////////////////////////////////////////////

/*
    Input : "Marvellous Multi OS"
    Output : 4
*/

#include<stdio.h>

int CountCapital(char *str)
{
    int iCount = 0;

    if(str == NULL)
    {
        printf("Invalid input ");
        return -1;

    }

    while(*str != '\0')
    {
        if((*str >= 'A') && (*str <= 'Z'))
        {
            iCount++;
        }
        str++;

    }

    return iCount;
}

int main()
{
    char arr[20];
    int iRet = 0;

    printf("Enter string\n");
    scanf("%[^'\n]s",arr);

    iRet = CountCapital(arr);

    printf("%d\n",iRet);
    return 0;
}

File: Assignment_55/program1.java
//  Write Java program which accept array of characters from user and 
// replace each capital character with its corresponding small character. 
/*
    Input :  b N j B R b A d G G  
    Output : b n j b r b a d g g
 */

////////////////////////////////////////////////////////////////////////////////
// File Name   :  program1.java
// Description :  Accept array of characters from user and replace each capital character
//                with its corresponding small character       
// Autor       :  Ankita Anil Patil
// Date        :  13/09/2025
///////////////////////////////////////////////////////////////////////////////


import java.util.*;

class MyArray 
{ 
        public void ArrayReplace(char arr[]) 
        { 
            int i = 0;
            for(i = 0; i < arr.length; i++ )
            {
                if(arr[i] >= 'A' && arr[i]<= 'Z')
                {
                    arr[i] = (char) (arr[i] + ('a' - 'A'));
                }
            }

            System.out.println("After replace each capital character with its corresponding small character");

            for(char ch : arr)
            {
                System.out.print(ch + "\t");
            }
            
        }
} 
public class program1
    
{   
    public static void main(String[] args) {
        

    Scanner sobj = new Scanner(System.in);

    System.out.println("Enter size of array");
    int iSize = sobj.nextInt();


    char[] Arr = new char[iSize];
    System.out.println("Enter charcater : ");

    int i = 0;
    
    for(i = 0; i < Arr.length; i++)
    { 
       Arr[i] = sobj.next().charAt(0);
    }

    MyArray mobj = new MyArray();
    mobj.ArrayReplace(Arr);
 
}
}
